addons/planner/static/src/js/planner.js

```diff
@@ -64,8 +64,9 @@
         setup: function(planner){
             var self = this;
             this.planner = planner;
+            this.dialog && this.dialog.destroy();
             this.dialog = new instance.planner.PlannerDialog(this, planner);
-            this.$(".oe_planner_progress").tooltip({html: true, title: this.planner.tooltip_planner, placement: 'bottom', delay: {'show': 500}});
+            this.$(".oe_planner_progress").tooltip({html: true, placement: 'bottom', delay: {'show': 500}}).attr('data-original-title', this.planner.tooltip_planner);
```

Comment:
If more than one planner are installed at same time, then one planner tool tip is clash with other planner tool tip. By setting 'original-title', tool tips doesn't clash.